src/main/webapp/script.js

```diff
@@ -22,3 +21,129 @@ function getDatabases(){
   });
 }
 
+function submitDatabaseForm() { 
+  document.getElementById("database-select-form").submit();
+}
+
+//Get database and selected tables from query string and /tables-from-db
+function getDatabaseAndTable(){
+    const tablesUrl = '/tables-from-db';
+    const search = window.location.search;
+    const queryString = tablesUrl + search;
+    var startIndex = 0;
+    var databaseString= "";
+    
+    for (var i = 0; i<search.length; i++){
+        if (search.charAt(i) == '='){
+            startIndex = i;
+        }
+        else if (search.charAt(i) == '&'){
+            break;
+        }
+        else{
+             databaseString += search.charAt(i);
+        }
+    }
+
+    databaseString = databaseString.substring(startIndex);
+    fetch(queryString)
+    .then(response => response.json())
+    .then((list) => { 
+        const DbText = document.createElement('p');
```

Comment:
I think constants should start with a lowercase letter, but correct me if I'm wrong!

Reply:
Can't believe I missed this, fixed!